ch.html

```diff
@@ -179,21 +170,14 @@ <h2 class="major"><font style="color:#ab2548;">/ </font>Who can enter?</h2>
                     <img alt="" class="image" src="images/ajandek0.png"/>
                 </section>
                 <div class="content">
-                    <h2 class="major"><font style="color:#ab2548;">/ </font>Prizes</h2>
+                    <h2 class="major"><font style="color:#ab2548;">/ </font>Preise</h2>
                     <p>
-                        Preventing a cyberattack involves collaboration, and all participants will receive a certificate
-                        in acknowledgement of their contribution. Courtesy of our generous sponsors, the best performing
-                        teams will have the chance to win valuable prizes.<br>
-                        All teams will receive a bronze, silver, gold or platinum rating. Those with a gold or platinum
-                        rating will be admitted to the virtual Alpha Challenge programming community. Any child
-                        (participants aged 8-16) in a platinum team, who constitute the crème-de-la-crème of ethical
-                        hackers, will be entered into a prize draw with the chance to win an X-Box Series S 512 GB
-                        console, Airpods 3 wireless earphones and a Samsung Galaxy Watch 4 smartwatch. Prizes will be
-                        shipped to winners, after verification of the contact details provided during registration.
+                        Die Verhinderung eines Cyberangriffs erfordert Zusammenarbeit, und alle TeilnehmerInnen erhalten ein Zertifikat als Anerkennung für ihren Beitrag. Dank unserer großzügigen Sponsoren haben die besten Teams auch noch eine Chance, wertvolle Preise zu gewinnen.<br>
```

Comment:
We don't have an unified formatting, but perhaps you could try autoformatting to make sure that we don't have very long lines. (IntellIJ does that by default, idk what you are using)

Reply:
I use VSCode currenty. I have word wrap enabled by default, so I didn't really notice. I added a max line length of 80 now to fix that.